app/views/lessons/show.html.erb

```diff
@@ -35,12 +35,12 @@
   <p><strong>Intro Tweet:</strong> <%= @lesson.intro %></p>
   <p><strong>Tweets: </strong></p>
   <div class="jumbotron">
-    <% @lesson.tweets.each do |tweet| %>
+    <% @lesson.tweets.each.order(:index) do |tweet| %>
```

Comment:
This isn't valid, since you can't apply an `order` operation after a `.each`. I'd suggest creating a variable `@ordered_tweets = @lesson.tweets.order(:index)`, then change this line to `@order_tweets.each do...`